x-pack/plugins/cases/public/components/recent_cases/index.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiText, EuiTitle } from '@elastic/eui';
+import React, { useMemo, useState } from 'react';
+
+import * as i18n from './translations';
+import { CaseDetailsHrefSchema, LinkAnchor } from '../links';
+import { RecentCasesFilters } from './filters';
+import { RecentCasesComp } from './recent_cases';
+import { FilterMode as RecentCasesFilterMode } from './types';
+import { useCurrentUser } from '../../common/lib/kibana';
+
+export interface RecentCasesProps {
+  allCasesHref: string;
+  createCaseHref: string;
+  getCaseDetailsHref: (caseDetails: CaseDetailsHrefSchema) => string;
+  goToAllCases: (ev: React.MouseEvent) => void;
+  onCaseDetailsNavClick: (caseDetails: CaseDetailsHrefSchema) => void;
+  perPage: number;
+}
+
+const RecentCases = ({
+  allCasesHref,
+  createCaseHref,
+  getCaseDetailsHref,
+  goToAllCases,
+  onCaseDetailsNavClick,
+  perPage,
```

Comment:
nit: `maxCases` or `totalCasesToShow` (you get the idea 😄 ) make more sense to me than `perPage` as the widget does not support pagination.